extension/entitystore/extension.go

```diff
@@ -94,9 +94,13 @@ func (e *EntityStore) Start(ctx context.Context, host component.Host) error {
 	e.serviceprovider = newServiceProvider(e.mode, e.config.Region, &e.ec2Info, e.metadataprovider, getEC2Provider, ec2CredentialConfig, e.done, e.logger)
 	switch e.mode {
 	case config.ModeEC2:
-		e.ec2Info = *newEC2Info(e.metadataprovider, e.done, e.config.Region, e.logger)
+		e.ec2Info = *newEC2Info(e.metadataprovider, e.kubernetesMode, e.done, e.config.Region, e.logger)
```

Comment:
how does the extension get that value?

Reply:
Are you asking about the `kubernetesMode`? it comes from the [translator](https://github.com/aws/amazon-cloudwatch-agent/blob/33ad73e6c22497166db2cab6359a2110b37de7de/translator/translate/otel/extension/entitystore/translator.go#L38) which gets passed as part of the config value